src/api/index.tsx

```diff
@@ -44,13 +46,14 @@ export const callApi = async ({
     headers.Authorization = `Bearer ${apiState.authToken}`;
   }
 
-  if (Object.keys(query).length) {
-    const queryString = Object.keys(query)
-      .map((k) => `${k}=${query[k]}`)
-      .join('&');
+  // Add the lang parameter to the querystring
+  const queryWithLang = Object.assign({ lang }, query);
```

Comment:
How about:

```js
const queryWithLang = {
   lang,
   ...query,
};
```

Reply:
This is nice, but I reversed the order, so that `lang` will override anything in `query`.